### DISCUSSION TITLE
how to get rid of huge empty space above heading of each slide

### DISCUSSION THREAD

**Discussion by zeltak** [quarto-dev/quarto-cli#1784] [Q&A] [answered]
Hi all

im currently getting huge empty spaces above the heading of each slide, see:

![CleanShot 2022-08-10 at 15 22 11](https://user-images.githubusercontent.com/498410/183899959-a539fb8c-8f08-49a8-9b0b-c71dd107069b.png)
what's (if at all possible) the way to make the header start at the top of the slide itself?

thx so much in advance

Z

**Comment by mcanouil** [comment]
Don't use level 1 header which produces the following.
<img width="600" alt="Screenshot of a revealjs slide with level 1 header, which is a big text vertically centered" src="https://user-images.githubusercontent.com/8896044/183936936-638f54d6-5c34-4547-9e9e-1bb57028e330.png">

While classical slides use level 2 headers (see <https://quarto.org/docs/presentations/revealjs/#creating-slides>).
<img width="600" alt="Screenshot of a revealjs slide with level 2 header, which is a big text left-aligned to the top of the slide" src="https://user-images.githubusercontent.com/8896044/183937275-860b3734-60b3-4285-9db5-043bfa05b53d.png">

**Reply by zeltak** [reply]
thx so much @mcanouil !

it seems to me that all header levels here produce this empty space

ie

```
## ENGLISH! {background-color="red"}

test

```

still has huge space on top

![CleanShot 2022-08-10 at 18 30 35](https://user-images.githubusercontent.com/498410/183945012-81341d99-5e00-496b-9084-0432b2cbe322.png)


thx again!

Z

**Reply by mcanouil** [reply]
Could you provide code instead of screenshots? A full small and reproducible example would be nice.

**Comment by zeltak** [comment]
sure sorry about that @mcanouil 

even using the Rstudio built in template I get the same space across all slides

example code

```

---
title: "Untitled"
format: revealjs
editor: visual
---

## Quarto

Quarto enables you to weave together content and executable code into a finished presentation. To learn more about Quarto presentations see <https://quarto.org/docs/presentations/>.

## Bullets

When you click the **Render** button a document will be generated that includes:

-   Content authored with markdown
-   Output from executable code

## Code

When you click the **Render** button a presentation will be generated that includes both content and the output of embedded code. You can embed code like this:

```{r}
1 + 1
```

```

**Reply by mcanouil** [reply, marked as answer]
I think you are getting confused by the fact that a reveal.js presentation fit the whole area available in a browser, while the slide keep the aspect ratio.
You can see on the screenshot below that the title is at the top left corner without spaces in the slide (it's from the code you provided).
<img width="1503" alt="image" src="https://user-images.githubusercontent.com/8896044/183958133-f481b0ee-1da1-47a0-9a98-06d50da03061.png">

PS: also use the Q&A category to help others find the questions and answers quicker.

**Comment by zeltak** [comment]
got it, thx!!
